core-web/libs/data-access/src/lib/dot-properties/dot-properties.service.spec.ts

```diff
@@ -64,12 +67,90 @@ describe('DotPropertiesService', () => {
 
         service.getKeys(keys).subscribe((response) => {
             expect(response).toEqual(apiResponse.entity);
+            done();
         });
         const req = httpMock.expectOne(`/api/v1/configuration/config?keys=${keys.join()}`);
         expect(req.request.method).toBe('GET');
         req.flush(apiResponse);
     });
 
+    describe('getKey', () => {
+        it('should return the value of a key if it exists in featureConfig', (done) => {
+            const key = 'existingKey';
+            const value = 'existingValue';
+            service.featureConfig = { [key]: value };
+            service.getKey(key).subscribe((result) => {
+                expect(result).toEqual(value);
+                done();
+            });
```

Comment:
Should we assert here that the http request actually didn't happened?

Reply:
I just added a test for this case